src/resources.ts

```diff
@@ -218,6 +218,7 @@ const otherResources: Resource[] = [
     "instant_skipAutomaticOptimizations",
     "Do not conduct automatic optimization of the route"
   ),
+  new Resource("instant_saveCandySword", "Do not use Candy Cane Sword Cane's Stab and Slash."),
```

Comment:
I don't think we need to add this as a pref, unless I'm not aware of a niche aftercore usage of the skills (they seem pretty mediocre in aftercore)?

Reply:
The pref is because Peppermint Rush is -10% Mys. There may be a That Guy for whom that breaks their Mys test.